StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -199,7 +189,6 @@ class RowButton: UIView {
 
         for view in [radioButton, imageView, labelsStackView, defaultBadge].compactMap({ $0 }) {
             view.translatesAutoresizingMaskIntoConstraints = false
-            view.isUserInteractionEnabled = false
```

Comment:
Interesting, won't this make the image view, labels, and badge intercept taps that should go to the RowButton?

Reply:
They get the tap first, but they don't do anything with it, so it goes to their parent view. Once you understand the responder chain this will all make sense!